hyperlane-mailbox/src/main.sw

```diff
@@ -22,9 +23,15 @@ const VERSION: u8 = 0;
 // "fuel" in bytes
 const LOCAL_DOMAIN: u32 = 0x6675656cu32;
 
+const ZERO_ID: ContractId = ContractId {
```

Comment:
Actually I think we can do https://github.com/FuelLabs/sway/blob/master/sway-lib-std/src/constants.sw#L5
Either to use `BASE_ASSET_ID` (which feels like it's named for a different purpose, to signify usage of the native fuel token...) or to use the `ContractId::from` syntax

Reply:
we can't use `ContractId::from` syntax in storage initializer
https://github.com/FuelLabs/sway/issues/2647